Source/Core/Core/HW/DSPHLE/UCodes/AESnd.h

```diff
@@ -45,10 +45,14 @@ class AESndUCode final : public UCodeInterface
   // https://github.com/extremscorner/libogc-rice/commit/cfddd4f3bec77812d6d333954e39d401d2276cd8
   // https://github.com/extremscorner/libogc2/commit/89ae39544e22f720a9c986af3524f7e6f20e7293
   static constexpr u32 HASH_2020_PAD = 0xa02a6131;
-  // July 19, 2022 version (padded to 0x0400 bytes) - fixed MAIL_TERMINATE. This is not currently
-  // included in libogc, only in libogc2 and libogc-rice (which generate a padded header file).
+  // July 19, 2022 version (padded to 0x0400 bytes) - fixed MAIL_TERMINATE. This padded version
+  // is only in libogc2 and libogc-rice (which generate a padded header file).
   // https://github.com/extremscorner/libogc2/commit/38edc9db93232faa612f680c91be1eb4d95dd1c6
   static constexpr u32 HASH_2022_PAD = 0x2e5e4100;
+  // March 13, 2023 version (0x03e8 bytes) - fixed MAIL_TERMINATE. This is the same fix as the
+  // above version, and was released in regular libogc 2.4.0  on April 17, 2023.
```

Comment:
Did you mean to put two spaces here?
```suggestion
  // above version, and was released in regular libogc 2.4.0 on April 17, 2023.
```

Reply:
Nope, fixed.